scripts/tasks/webpack.ts

```diff
@@ -9,8 +9,9 @@ type JustArguments<T extends Record<string, unknown>> = Arguments & T;
 export function webpack() {
   const args: JustArguments<Partial<{ production: boolean }>> = argv();
   return webpackCliTask({
-    webpackCliArgs: args.production ? ['--production'] : [],
+    webpackCliArgs: args.production ? ['--mode=production'] : [],
```

Comment:
I am assuming this works as is, but in the docs it had double quotes around "production".

Reply:
Yeah so funny thing...at least when invoking the CLI via spawnSync it *doesn't* work with the quotes 🙄